packages/stellar-axelar-std-derive/src/testdata/storage_schema_generation_succeeds.golden

```diff
@@ -0,0 +1,141 @@
+#[contracttype]
+enum DataKey {
+    Counter,
+    Message(Address),
+    LastCaller(u64),
+    Flag(String, Address),
+    OptionalMessage(u32),
+}
+impl DataKey {
+    pub fn counter(env: &soroban_sdk::Env) -> Option<u32> {
```

Comment:
it's unnecessary to implement these methods on DataKey since they're never used directly on it. We can just generate the root `counter` getter with this implementation